.github/workflows/backend-typing.yml

```diff
@@ -0,0 +1,82 @@
+name: backend typing
+on:
+  push:
+    branches:
+      - master
+      - releases/**
```

Comment:
Do we still use `releases` branches?

Reply:
I'm not sure. I copied this from the snuba test workflow.

Reply:
```suggestion
```